packages/orm/src/drivers/PostgresDriver.ts

```diff
@@ -52,13 +52,15 @@ export class PostgresDriver implements Driver {
     parsed: ParsedFindQuery,
     settings: { limit?: number; offset?: number },
   ): Promise<any[]> {
-    const knex = this.getMaybeInTxnKnex(em);
-    return buildKnexQuery(knex, parsed, { limit: em.entityLimit, ...settings });
+    const { sql, bindings } = buildRawQuery(parsed, { limit: em.entityLimit, ...settings });
+    return this.executeQuery(em, sql, bindings);
```

Comment:
I couldn't find an easy way to prefix our existing `buildKnexQuery` with "a blob of CTE sql", because: a) you can either use knex.raw or knex.from but not both together (afaict) and b) their CTE support wants to be fluent DSL like `knex.withRecursive(etc => ...)` and I just wanted to hand it the raw sql.

So, I went ahead and moved this over to a `buildRawQuery` that creates our SQL strings directly, without going through knex builders.

Thanks to the AST-ification of `ParsedFindQuery`, this was actually surprisingly easy to do; I'd been meaning to do it for awhile anyway, to decouple us from knex, and make it easier to explore alternative drivers.